patch_all.py

```
@@ -0,0 +1,73 @@
+                                    )));
+                                }
+                            }
+                        }"""
+
+content = content.replace(target, replacement)
+
+with open('crates/logfwd-config/src/validate.rs', 'w') as f:
+    f.write(content)
+
+# Fix logfwd-io
+with open('crates/logfwd-io/src/macos_log_input.rs', 'r') as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)